webui/react/src/services/apiConfig.ts

```diff
@@ -1775,17 +1775,23 @@ export const launchTensorBoard: DetApi<
 
 export const getJobQueue: DetApi<
   Service.GetJobQParams,
-  Api.V1GetJobsResponse,
+  Api.V1GetJobsV2Response,
   Service.GetJobsResponse
 > = {
   name: 'getJobQ',
   postProcess: (response) => {
-    response.jobs = response.jobs.filter((job) => !!job.summary);
+    const internalResponse: Service.GetJobsResponse = {
+      jobs: [],
+      pagination: response.pagination,
+    };
+    internalResponse.jobs = response.jobs
+      .map((jobPack) => (jobPack.full || jobPack.limited) as Type.Job)
+      .filter((job) => !!job.summary);
```

Comment:
**super nit:** can the filter run first so mapping will be a smaller list to run through (very very trivial perf diff for sure).